operator/roles/ansible-service-broker/tasks/main.yml

```diff
@@ -3,7 +3,12 @@
 # Only want to do this once per execution of role
 - name: Get cluster api_groups
   set_fact:
-    api_groups: "{{ lookup('k8s', cluster_info='api_groups', kubeconfig=lookup('env', 'K8S_AUTH_KUBECONFIG')) }}"
+    api_groups: "{{ lookup('k8s', cluster_info='api_groups', kubeconfig=lookup('env', 'K8S_AUTH_KUBECONFIG', errors='ignore')) }}"
```

Comment:
I don't believe that this one would ever fail because of RBAC. @fabianvf may know better but I don't think this every fails because of RBAC problems.

Reply:
done